test/fft_utest.cpp

```diff
@@ -140,6 +143,67 @@ class FftTest : public ::testing::Test {
             ASSERT_EQ(ifft_1, v);
         }
     }
+
+    /** Convert a number into a vector of digits padded with zeros
+     *
+     * @param vec destination vector
+     * @param n vector size
+     * @param num number in string representation
+     */
+    void big_num_to_vec(vec::Vector<T>& vec, int n, std::string num)
+    {
+        int i;
+        int len = num.length();
+        for (i = 0; i < len; i++) {
+            vec.set(i, num[len - i - 1] - '0');
+        }
+        for (; i < n; i++) {
+            vec.set(i, 0);
+        }
+    }
+
+    /** Compute sum of 2 large numbers in string representation
+     *
+     * @param str1 first string
+     * @param str2 second string
+     *
+     * @return the sum string
+     */
+    std::string big_num_add(std::string& str1, std::string& str2)
+    {
+        if (str1.length() > str2.length()) {
+            std::swap(str1, str2);
+        }
+
+        std::string str;
+
+        int n1 = str1.length();
```

Comment:
Return type of `std::string.length` is `size_t` not `int` (I think I mentioned it in a previous review round)